.scrutinizer.yml

```diff
@@ -0,0 +1,10 @@
+filter:
+    excluded_paths:
+        - 'Tests/*'
+        - 'vendor/*'
+tools:
+    external_code_coverage: true
+    php_analyzer: true
+    sensiolabs_security_checker: true
+checks:
+    php: true
```

Comment:
should be removed. I'm already managing the scrutinizer config inside scrutinizer itself